game/scripts/vscripts/heroes/vengefulspirit/command_aura.lua

```diff
@@ -0,0 +1,165 @@
+---@class vengefulspirit_command_aura_lua : CDOTA_Ability_Lua
+vengefulspirit_command_aura_lua = class({})
+
+---@class modifier_vengefulspirit_command_aura_lua : CDOTA_Modifier_Lua
+modifier_vengefulspirit_command_aura_lua = class({})
+LinkLuaModifier( "modifier_vengefulspirit_command_aura_lua", LUA_MODIFIER_MOTION_NONE )
+
+---@class modifier_vengefulspirit_command_aura_effect_lua : CDOTA_Modifier_Lua
+modifier_vengefulspirit_command_aura_effect_lua = class({})
+LinkLuaModifier( "modifier_vengefulspirit_command_aura_effect_lua", LUA_MODIFIER_MOTION_NONE )
+
```

Comment:
arrange file in blocks, first linkluamodifiers, then each modifier plus its functions, ability last:
```
LinkLuaModifiers()

--- Command aura
modifier_vengefulspirit_command_aura = class({})

function modifier_vengefulspirit_command_aura:IsHidden()
    return true
end

...

--- Command aura effect
modifier_vengefulspirit_command_aura_effect = class({})

function modifier_vengefulspirit_command_aura_effect:IsHidden()
    return true
end

...

--- Vengeful Spirit: Command aura
vengefulspirit_command_aura_lua = class({})

...
```